sonic-net/sonic-buildimage
# [Platform/Seastone2] add optoe based sfp implementation

#### Why I did it
Add sfp-refactor optoe based sfp implementation for Seastone2

#### How I did it
Inherit and implement the sfp api according to new sfp-refactor
architecture

#### How to verify it
Verify sfp status and sfp configure commands on seastone2 with new
implementation

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog

#### Link to config_db schema for YANG module changes

#### A picture of a cute animal (not mandatory but encouraged)
